security/vault.go

```diff
@@ -190,6 +190,7 @@ func (v *Vault) encrypt(data []byte) (string, error) {
 		// User has deleted all the secrets. the file will be empty.
 		return "", nil
 	}
+	data = append(data, []byte("\nGAIA_CHECK=!CHECK_ME!")...)
```

Comment:
That looks a bit magic to me. What do you think about moving this to constant variable?

Reply:
Sure thing! 😊